src/components/GlobalPreferences/Network/Network.js

```diff
@@ -33,15 +33,15 @@ function Network({ wrapper }) {
   return (
     <React.Fragment>
       <Box heading="Node settings">
-        <Label>
+        <Label theme={theme}>
```

Comment:
We could also update `Label` so it uses `theme` internally:

```jsx
function Label(props) {
  const theme = useTheme()
  return (
    <label
      {...props}
      css={`
        color: ${theme.contentSecondary};
        display: block;
        margin-bottom: ${2 * GU}px;
      `}
    />
  )
}
```

Reply:
Sure, let's follow this approach for apps to have access to the theme.